src/openrct2/world/Map.cpp

```diff
@@ -706,12 +706,8 @@ int32_t map_height_from_slope(const CoordsXY coords, int32_t slope, bool isSlope
 
 bool map_is_location_valid(const CoordsXY coords)
 {
-    if (coords.x < (MAXIMUM_MAP_SIZE_TECHNICAL * 32) && coords.x >= 0 && coords.y < (MAXIMUM_MAP_SIZE_TECHNICAL * 32)
-        && coords.y >= 0)
-    {
-        return true;
-    }
-    return false;
+    return coords.x < (MAXIMUM_MAP_SIZE_TECHNICAL * 32) && coords.x >= 0 && coords.y < (MAXIMUM_MAP_SIZE_TECHNICAL * 32)
+        && coords.y >= 0;
```

Comment:
Old notation is easier to read.

Reply:
I would even suggest to split this into multiple tests.

Reply:
done